docs/Actions/load-user.md

```diff
@@ -0,0 +1,93 @@
+---
+id: LoadUser
+title: Load User
+sidebar_label: Load User
+---
+
+An action that allows altering the user which the subsequent actions refer to. The user change affects all actions after the `Load User` action and after all actions have finished executing the user will be reverted back to the initial one. By default, all actions that need an user for their execution will refer to the currently logged in user.
+
+This action also supports loading multiple users by providing multiple user identifiers. The last user identifier provided will also be used as the context user under which all actions execute. Loading multiple users creates a hidden context entity that is leveraged by the following actions which support this:
+
+- [Grant Role](/docs/Actions/grant-role.md)
+- [Revoke Role](/docs/Actions/revoke-role.md)
+- [Authorize User](/docs/Actions/authorize-user.md)
+- [Unauthorize User](/docs/Actions/unauthorize-user.md)
+- [Delete User](/docs/Actions/delete-user.md)
+- [Send Email](/doc/Actions/send-email.md).
+
+When used in conjunction with actions such as [Grant Role](/docs/Actions/grant-role.md), the `Grant Role` action will grant the role specified to all users loaded by this action.
+
+## `Typical Use Cases`
+
+- Impersonating an user
+- Loading the user before granting the necessary roles
+- Loading users before sending a mass email
+
+## `Don't use it to`
+
+- Load multiple users by a specific condition, use [Load Users From SQL](/docs/actions/load-users-sql.md) instead
+
+## `Related Actions`
+
+- [Grant Role](/docs/Actions/grant-role.md) - grants the users in the context the specified roles
+- [Send Email](/doc/Actions/send-email.md) - sends an email to the users loaded in the context
+
+## `Input Parameter Reference`
+
+| Parameter     | Description                           | Supports Variables |
+|---------------|---------------------------------------|--------------------|
+| User Identifier | The identifier which will indicate which user to load into context. You can use the user ID, username or the email as identifiers. <br/> You can load multiple users by providing multiple values separated by a comma. | Yes |
+| Portal ID | The identifier which will specify in which portal to look for the user indicated by the 'User Identifier'. If left blank, the action will search for the user across all portals. Valid values are either the portal ID or the portal name. | Yes |
+
+## `Examples`
+
+### `1. Load an user and grant it the Subscribers role`
+
+The actions below will load the user with the identifier (user email) 'john.doe@gmail.com' into the context and then grant him the role `Subscribers` with an indefinite expiration date.
+
+```json
+{
+    "Title": "Load User",
+    "ActionType": "LoadUser",
+    "Description": "Load the user into the context",
+    "Condition": null,
+    "Parameters": {
+        "Id": "john.doe@gmail.com",
+        "Portal": ""
+    }
+}
+```
+
```

Comment:
Maybe we can add an Update User Profile Example for FirstName and LastName, Something simple